chalice/deploy/validate.py

```diff
@@ -7,6 +7,34 @@
 from chalice.config import Config  # noqa
 
 
+class ExperimentalFeatureError(Exception):
+    _ERROR_MSG = (
```

Comment:
Thoughts on putting this in constants with (some) of our other error messages?

Reply:
Don't really have a preference, we're not super consistent with this (some of the deployer error messages are in deployer.py).